### PR TITLE
Document engineering leadership team

### PR DESCRIPTION
**Pull request by nicksnyder**
I drafted a vision statement and wrote down some goals (which are hopefully not controversial since we have started working on most of them and have already started discussing the others). I view this as a starting point that I would like to merge by the end of the week and we can iterate from there. This also serves as an illustrative example for how I think about creating/formatting/communicating goals.

### CODE REVIEW

File: handbook/engineering/leadership/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#### Fullstack engineer role
+
+- Problem: Cloud and Code Intelligence teams have a mix of frontend and backend work, but current workload is skewed toward the latter. It doesn't make sense to hire a single frontend dev because they would end up working alone and there might not be enough work to sustain a full time frontend dev.
+- Plan: We are defining a new fullstack role and interview process to help us find engineers who can do the necessary frontend work, but also contribute to our backend systems. We think this role could be valuable for other teams, like Search and Campaigns, in the future. 
+- Owners: Tomás, Aida
+- Status: [In progress](https://github.com/sourcegraph/interviews/pull/136).
+
+#### Engineering manager onboarding
+
+- Problem: We recently hired and onboarded our first group of engineering managers and onboarded each of them in an ad hoc way. We will be hiring at least one more engineering manager in the near future (for the web team) and we want to
```
Comment (unknwon) [inline handbook/engineering/leadership/index.md:44]: missing ending?

**Review by unknwon — APPROVED** [review]
❤️ Thanks for adding this, looks great to me!

**Review by asdine — APPROVED** [review]
Looks good 👍

**Review by pecigonzalo — APPROVED** [review]

**Review by aidaeology — APPROVED** [review]
LGTM

File: handbook/engineering/web/index.md [hunk 2]
```diff
@@ -77,10 +77,11 @@ Before web team syncs, teammates and stakeholders should write down under "Discu
 
 ## Members
 
-- [Loïc Guychard](../../../company/team/index.md#loic-guychard) ([engineering manager](../roles.md#engineering-manager))
-- [Felix Becker](../../../company/team/index.md#felix-becker)
-- [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
-- [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
+- We're hiring a [Product Manager](../../product/roles/product_manager.md) for this role. [Christina Forney](../../../company/team/index.md#christina-forney-she-her) is involved in the meantime.
```
Comment (felixfbecker) [inline handbook/engineering/web/index.md:80]: Don't we have J. starting in 2 weeks?

**Review by felixfbecker — APPROVED** [review]

File: handbook/engineering/web/index.md (hunk 2 above)
Comment (nicksnyder) [reply handbook/engineering/web/index.md:80]:
```suggestion
- J.K. starting 2020-09-14 ([Product Manager](../../product/roles/product_manager.md))
```

File: handbook/engineering/leadership/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- Owner: Nick
+- Plan: Document the onboarding process for engineering managers in our handbook before the EM for the web team starts.
+- Status: Not started.
+
+### Medium term (~3-6 months)
+
+#### Publicly documented compensation bands
+
+- Problem: When we were smaller, it was easy to contain consistent compensation across the entire team because there were fewer people to coordinate with. Now that we are bigger and have more engineering managers making offers and compensation decisions, we need to have clearly documented compensation bands so we can maintain consistency while federating decision making. We want to make this public because it will help candidates understand what they can expect before they even apply. This is a better candidate experience and it avoid rare but real situations where a candidate goes through our interview process only to discover that their compensation expectations don't fit into our bands.
```
Comment (chrispine) [inline handbook/engineering/leadership/index.md:54]: s/avoid/avoids/

File: handbook/engineering/leadership/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  2. Update the [compensation documentation in our handbook](../../people-ops/compensation.md#ranges) so candidates and current teammates know what our bands are.
+- Status: [First milestone is in progress](https://docs.google.com/spreadsheets/d/1iQsfchsnY5CxKSZfcnazpnJ1uSL6-gMuEI-wgsfLq3I/edit#gid=1795648823).
+
+#### Publicly documented interview modules
+
+- Problem: Any candidate who has interviewed at multiple companies knows that interview processes are hit or miss. We have spent time and effort to ensure that we have a great interview process, and candidates regularly tell us that we do after going through it. Right now, candidates don't know what to expect out of our interview process beyond the title of each interview module and a brief description.
+- Plan: We want candidates to know up front as much as possible about each interview, including exact questions that we will be asking them as and how we will be evaluating their answers (as much as possible). Sharing this information would have many benefits:
```
Comment (chrispine) [inline handbook/engineering/leadership/index.md:64]: s/them as/them/

File: handbook/engineering/leadership/index.md (hunk 1 above)
Comment (nicksnyder) [reply handbook/engineering/leadership/index.md]: Thanks for pointing out, fixed!

File: handbook/engineering/leadership/index.md (hunk 3 above)
Comment (nicksnyder) [reply handbook/engineering/leadership/index.md:54]: Thanks! Fixed in 35c018d22209f2625d23b8ec330ad56699b66352

File: handbook/engineering/leadership/index.md (hunk 4 above)
Comment (nicksnyder) [reply handbook/engineering/leadership/index.md:64]: Thanks! Fixed in 35c018d22209f2625d23b8ec330ad56699b66352

**Pull request merged**